test/Service/ConsoleViewHelperManagerDelegatorFactoryTest.php

```diff
@@ -83,10 +85,16 @@ public function testCreateUrlHelperFactoryInjectsHelperWithRouterAndRouteMatchWh
         $r = new ReflectionMethod($this->factory, 'createUrlHelperFactory');
         $r->setAccessible(true);
         $factory = $r->invoke($this->factory, $container->reveal());
-
         $helper = $factory();
 
-        $this->assertAttributeSame($router->reveal(), 'router', $helper);
-        $this->assertAttributeSame($routeMatch->reveal(), 'routeMatch', $helper);
+        $reflectionClass = new \ReflectionClass(get_class($helper));
+        $reflectionPropertyRouter = $reflectionClass->getProperty('router');
+        $reflectionPropertyRouter->setAccessible(true);
+
+        $reflectionPropertyRouteMatch = $reflectionClass->getProperty('routeMatch');
+        $reflectionPropertyRouteMatch->setAccessible(true);
+
+        $this->assertEquals($router->reveal(), $reflectionPropertyRouter->getValue($helper));
```

Comment:
These should be `assertSame` rather than `assertEquals`

Reply:
updated.